README.md

```diff
@@ -4,4 +4,4 @@
 
 If you need help with using GitHub, let me know. - Moses
 
-
+Just to make a note here, make sure you guys clone the files into a new branch before you edit the files. After you make changes and test your code, make a pull request for the file you have changed instead of just merging with the master branch. If you have any questions, you can ask me or try to find something on Google, there is a lot out there teaching Git. -Christian
```

Comment:
Thanks for the info. I believe I have to accept the request in order to merge with the master. Before merging anything I will be confirming with you guys. - Moses